packages/website/src/pages/wiki/api-docs.astro

```diff
@@ -177,7 +177,7 @@ const flash = handleFlash(Astro)
 	<p class='mb-2'>Each locale has a list of pronouns set available to it.</p>
 	<div class='flex flex-none gap-4 items-center mb-4 px-4 py-2 font-semibold border rounded text-blue-600 border-blue-600 dark:text-blue-400 dark:border-blue-400'>
 		<div class='w-6 h-6 shrink-0' set:html={infoIcon}/>
-		<p>Only the "en" locale is supported at this time.</p>
+		<p>Only the "en", "nb", and "nn" locales are supported at this time.</p>
```

Comment:
This would probably best be replaced by some other message, as this was intended to outline the fact while the structure seems to indicate there may be multiple languages, only ` en` is implemented.

The documentation should explain the canonical behavior for dealing with sets in multiple languages, what to pick etc. which is currently undefined as I haven't got around experimenting with it